pkg/operator/operator_test.go

```diff
@@ -446,6 +446,20 @@ func TestUpgradeableStatus(t *testing.T) {
 	}
 }
 
+func isDegraded(r runReport) bool {
```

Comment:
(nit) this could be turned into a test helper

```suggestion
func assertDegraded(t *testing.T, r runReport) {
    t.Helper()
    ...
}
```

Reply:
+1, I'll refactor the tests overall in a separate PR. There's a few more functions I'd like to move. Thanks for the tip!